### DISCUSSION TITLE
Very first release!

### DISCUSSION THREAD

**Discussion by picosam** [semantic-release/semantic-release#1810] [Q&A] [answered]
Hello, I just added semantic release to a new project, and I would like to run it for the first time without bumping the version, just to tag the repository with the current version and maybe generate the changelog file. What's the command for this?

**Comment by travi** [comment, marked as answer]
there is no command to execute for `semantic-release` to catch up to existing versions. you must manually [ensure that a properly formatted tag exists for the current version](https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags). once that is in place, `semantic-release` can take over for future releases.
